### PR TITLE
Add in my blog post about H.264

### PR DESCRIPTION
**Pull request by paultag**

### CODE REVIEW

**Comment by paultag** [comment]
https://github.com/KittyCAD/website/pull/1465 <-- related post, also after that and this land the submodule needs a bump

File: content/posts/fixing-an-h264-encoding-bug.mdx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+title: 'Fixing an H.264 encoding bug in the KittyCAD API'
+excerpt: |
+    Let's go through a bug deep within our WebRTC stream we solved this week!
+coverImage: 'meta-images/paultag-post-social.jpg'
+date: '2023-11-03T20:00:00Z'
+author:
+    name: Paul Tagliamonte
+    picture: '/paultag.jpg'
+ogImage:
+    url: 'meta-images/paultag-post-social.jpg'
```
Comment (paultag) [inline content/posts/fixing-an-h264-encoding-bug.mdx:11]: this doesn't exist yet, I didn't know how these were being generated

File: content/posts/fixing-an-h264-encoding-bug.mdx [hunk 2]
```diff
@@ -0,0 +1,143 @@
+---
+title: 'Fixing an H.264 encoding bug in the KittyCAD API'
+excerpt: |
+    Let's go through a bug deep within our WebRTC stream we solved this week!
+coverImage: 'meta-images/paultag-post-social.jpg'
+date: '2023-11-03T20:00:00Z'
```
Comment (paultag) [inline content/posts/fixing-an-h264-encoding-bug.mdx:6]: date is wrong natch

File: content/posts/fixing-an-h264-encoding-bug.mdx (hunk 1 above)
Comment (franknoirot) [reply content/posts/fixing-an-h264-encoding-bug.mdx:11]: I've had to make these manually for now (our auto-generated meta-images are still using out-of-date branding after the site flip), so I'll make this and add it to your website PR.

**Comment by franknoirot** [comment]
Do you mind if I write some alt text for your images? Especially the charts

**Comment by paultag** [comment]
@franknoirot that would be huge! I'd appreciate any help at all! Great call, super important.

**Pull request merged**